mono/metadata/icall.c

```diff
@@ -950,6 +952,21 @@ ves_icall_System_Array_GetRank (MonoObjectHandle arr, MonoError *error)
 	return result;
 }
 
+#ifdef ENABLE_NETCORE
+gint32
+ves_icall_System_Array_GetCorElementTypeOfElementType (MonoArrayHandle arr, MonoError *error)
```

Comment:
Ideally these should be intrinsics, or at least handle-less. I believe @EgorBo played with this one earlier.